internal/pkg/service/appproxy/http/router.go

```diff
@@ -153,19 +152,19 @@ func (r *Router) createProvider(providerConfig options.Provider, app DataApp) (o
 
 	proxyConfig, err := r.authProxyConfig(app, providerConfig)
 	if err != nil {
-		return provider, errors.Errorf("unable to create oauth proxy config for app %s %s: %w", app.ID, app.Name, err)
+		return provider, errors.PrefixErrorf(err, "unable to create oauth proxy config for app %s %s", app.ID, app.Name)
```

Comment:
Please wrap each argument in `"`, for example `%s` -> `"%s"`.

It may be very usefull "to see" the empty string, if there is a bug.